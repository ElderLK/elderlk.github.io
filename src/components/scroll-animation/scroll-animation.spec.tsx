import React from 'react'
import { render, screen } from '@testing-library/react'

import ScrollAnimation from './scroll-animation.component'

describe('<ScrollAnimation />', () => {
  it('should render the component', () => {
    const { container } = render(
      <ScrollAnimation>
        <ol>
          <li>one</li>
          <li>two</li>
          <li>three</li>
        </ol>
      </ScrollAnimation>
    )

    expect(screen.getAllByRole('listitem').length).toEqual(3)

    expect(container.firstChild).toMatchSnapshot()
  })
})
