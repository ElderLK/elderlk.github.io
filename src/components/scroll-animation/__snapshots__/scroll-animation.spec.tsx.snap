// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ScrollAnimation /> should render the component 1`] = `
<ol>
  <li>
    one
  </li>
  <li>
    two
  </li>
  <li>
    three
  </li>
</ol>
`;
